src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/ECDsa.cs

```diff
@@ -8,6 +8,20 @@ namespace System.Security.Cryptography
 {
     public abstract class ECDsa : AsymmetricAlgorithm
     {
+        public ECDsa() { }
+
+        public ECDsa(ECNamedCurve curve)
+        {
+            if (curve.Oid != null)
+            {
+                _curve = new Oid(curve.Oid.Value, curve.Oid.FriendlyName);
+            }
+        }
+
+        public abstract ECParameters ExportNamedCurveParameters(bool includePrivateParameters);
```

Comment:
@weshaggard While I assume we're allowed new abstract members in contract revisions; I further assume that we have to do virtual+throw in the implementation libraries for anything not in the contract revision that added the type.

Or is there some awesomeness that will let us do this in Core?